### ISSUE TITLE
Pressing "RB" on a XBox 360 gamepad freezes cfclient - "Exception: Connection timed out"

### ISSUE THREAD

**Issue by schlagenhauf** [bitcraze/crazyflie-clients-python#597]
Opening cfclient and pressing "RB" on my XBox 360 gamepad while not connected to a Crazyflie freezes the application and after about a minute, the following is printed in the terminal, before freezing again:

```
Traceback (most recent call last):
  File "/home/pntagrl/repos/gh/crazyflie-clients-python/src/cfclient/ui/tabs/FlightTab.py", line 638, in _assisted_control_updated
    self.helper.cf.param.set_value("flightmode.althold", str(enabled))
  File "/home/pntagrl/repos/gh/crazyflie-clients-python/venv/lib/python3.10/site-packages/cflib/crazyflie/param.py", line 320, in set_value
    raise Exception('Connection timed out')
Exception: Connection timed out
```

Looking at the used device mapping (`xbox360_mode1`), it seems that "RB" is bound to "Assisted control".

**Edit:**
When a CF is connected, pressing "RB" does not freeze the application anymore (and it looks like it affects the thrust, as intended?) but the following error is produced with every button press:

```
===== 2022.05.03 10:05:58 =====
Traceback (most recent call last):
  File "/home/pntagrl/repos/gh/crazyflie-clients-python/venv/lib/python3.10/site-packages/cflib/crazyflie/param.py", line 342, in set_value
    value_nr = int(value)
ValueError: invalid literal for int() with base 10: 'True'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/pntagrl/repos/gh/crazyflie-clients-python/src/cfclient/ui/tabs/FlightTab.py", line 638, in _assisted_control_updated
    self.helper.cf.param.set_value("flightmode.althold", str(enabled))
  File "/home/pntagrl/repos/gh/crazyflie-clients-python/venv/lib/python3.10/site-packages/cflib/crazyflie/param.py", line 344, in set_value
    value_nr = float(value)
ValueError: could not convert string to float: 'True'
```

**Debug info:**
Commit: 767b4d832e569d5ddf60ff104aabd1e85c9dad8b
Cfclient version: 2022.3.post14.dev0+767b4d8
System: linux
Python: 3.10.4
Qt: 5.15.2
PyQt: 5.15.6
cflib: 0.1.18.1 (from PyPI)
cfclient installed from source in virtual environment via `pip install -e .`

**Comment by knmcguire** [comment]
This seems to be related to the issue we are having here: https://github.com/bitcraze/crazyflie-clients-python/issues/587 

Are you using windows as well? 10 or 11?

**Comment by schlagenhauf** [comment]
I'm using Arch.

#587 reminded me that I had a corrupted config some days ago after cfclient crashed. However deleting `~/.config/cfclient` does not solve the above problem.

**Comment by knmcguire** [comment]
Ah oke... it seems that is probably something else. We have some QT issues recently but this seems to be something different.

Could you share more info of your dev environment? Like which python version you have and if this is the cfclient and cflib of the latest release or installed from source?

**Comment by schlagenhauf** [comment]
Of course, I added it to the issue text.

**Comment by knmcguire** [comment]
Ah interesting! we should indeed fix those issues. I've tried to fix these parts myself, but when rebuilding the cfclient, I still get the freezing with the controller, so there is probably something else going on.

**Comment by krichardsson** [comment]
I think this might be related to bitcraze/crazyflie-lib-python#341 and possibly fixed in bitcraze/crazyflie-lib-python#341 as a side effect. Would it be possible for you @schlagenhauf to re-test with the latest python lib?

**Comment by knmcguire** [comment]
ah actually @schlagenhauf you tried a version of the cfclient that didn't include this fix... so we are not sure what caused it :) No worries, good thing it is solved.
